### PR TITLE
New Feature: Add images for the wiki and edit links to point to it

### PR DESCRIPTION
**Pull request by Thomas-Boi**
# **New feature section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *New Feature: a brief description of feature*

## This PR adds/fixes

I transferred the content of `CONTRIBUTING.md` to the [Wiki tab](https://github.com/devicons/devicon/wiki). I also added some images and I'll add them to the wiki once the PR is merged.

## Notes

I think we should keep the `CONTRIBUTING` until next release. Then, we can switch over to the Wiki. Also, if it's not set up already, I think we should limit updating the Wiki to maintainers only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,12 +156,12 @@ Add css rules in your stylesheet
 
 <h2 id="request-icon">Requesting icon</h2>
 <p>
-    When you want to request a icon please feel feel to create a issue. See our <a href="/CONTRIBUTING.md#requestingIcon">contribution guidelines</a> for more information.
+    When you want to request a icon please feel feel to create a issue. See our <a href="https://github.com/devicons/devicon/wiki/Requesting-an-Icon">contribution guidelines</a> for more information.
```
Comment (Panquesito7) [inline README.md:159]:
```suggestion
    When you want to request an icon please feel free to create an issue. See our <a href="https://github.com/devicons/devicon/wiki/Requesting-an-Icon">Wiki</a> for more information.
```

File: README.md [hunk 2]
```diff
@@ -156,12 +156,12 @@ Add css rules in your stylesheet
...
 </p>
 
 <h2 id="contribute">Contributing</h2>
 <p>
-    We are happy for every contribution, whether it's new icons, features, or maintainers. Please have a look at our <a href="CONTRIBUTING.md">contribution guidelines</a> to see how you can contribute to this project.
+    We are happy for every contribution, whether it's new icons, features, or maintainers. Please have a look at our <a href="https://github.com/devicons/devicon/wiki">contribution guidelines</a> to see how you can contribute to this project.
```
Comment (Panquesito7) [inline README.md:164]:
```suggestion
    We are happy with every contribution, whether it's new icons, features, or maintainers. Please have a look at our <a href="https://github.com/devicons/devicon/wiki">Wiki</a> to see how you can contribute to this project.
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution and hard work! 😄👍

**Comment by maltejur** [comment]
The new wiki looks great, good job!

One small thing, I think it would be a good idea to add a custom sidebar because currently it looks a bit cluttered.

**Comment by Thomas-Boi** [comment]
> The new wiki looks great, good job!
> 
> One small thing, I think it would be a good idea to add a custom sidebar because currently it looks a bit cluttered.

That's a good idea. I'll work on it later this week

**Comment by Thomas-Boi** [comment]
I've added a sidebar to the wiki. I'll also merge this PR just so I have access to the images

**Pull request merged**